Core/OxidEeEvents.php

```diff
@@ -9,14 +9,15 @@
 
 namespace Wirecard\Oxid\Core;
 
-use Exception;
-
+use OxidEsales\Eshop\Core\Field;
```

Comment:
With the refactoring the line 18,19,20 are not used anymore.

Reply:
Which refactoring are you referring to? `Helper`, `DatabaseHelper` and `PaymentMethodFactory` are all still in use.

Reply:
I got the result from my IDE, but as i see it the Helper is in the same folder, then you don't need to add the use. Same for the other two. I see this as a common error. Classes witch are in same folder don't need an use.

Reply:
I see. We might want to add a PHPCS sniff for that in the future.